Platon/backend/app/profile_management/views.py

```diff
@@ -107,6 +111,170 @@ def delete(user_id,self):
         else:
             return make_response(jsonify({'error':'Wrong input format'}),400)
 
+    @profile_management_ns.route("/jobs")
+    class JobsAPI(Resource):
+
+        """
+            Returns all job names
+        """
+        @api.doc(responses={200: 'Valid Response', 404: 'Empty List', 500: 'Database Connection Problem'})
+        def get(self):
+            try:
+                jobs = Jobs.query.all()
```

Comment:
I think your code looks fine. But please add some commands which says the functionality of all endpoints for swagger. You can find some exples in the previous codes.

Reply:
I added them.